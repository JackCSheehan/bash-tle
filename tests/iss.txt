ISS (ZARYA)             
1 25544U 98067A   22255.73370370  .00008229  00000+0  15112-3 0  9999
2 25544  51.6420 262.8647 0002292 226.1291 116.9782 15.50169761358756
