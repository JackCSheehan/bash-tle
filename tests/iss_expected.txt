ISS (ZARYA),25544,U,98,067,A,22,255.73370370,.00008229,00000+0,15112-3,0,999,9,25544,51.6420,262.8647,0002292,226.1291,116.9782,15.50169761,35875,6
